flask/app/participants.py

```diff
@@ -81,23 +108,26 @@ def list_participants():
                 == models.users_groups_table.columns.group_id,
             )
             .filter(models.users_groups_table.columns.user_id == user_id)
-            .order_by(column)
-            .limit(max_rows)
+            .order_by(order())
+            .limit(limit)
+            .offset(offset)
         )
+
     else:
-        participants = (
-            models.Participant.query.options(
-                joinedload(models.Participant.family),
-                joinedload(models.Participant.tissue_samples).joinedload(
-                    models.TissueSample.datasets
-                ),
-                joinedload(models.Participant.created_by),
-                joinedload(models.Participant.updated_by),
-            )
-            .filter(models.Participant.participant_codename.like(starts_with))
-            .order_by(column)
-            .limit(max_rows)
+        participants = models.Participant.query.options(
+            joinedload(models.Participant.family),
+            joinedload(models.Participant.tissue_samples).joinedload(
+                models.TissueSample.datasets
+            ),
+            joinedload(models.Participant.created_by),
+            joinedload(models.Participant.updated_by),
         )
+        for f in filt:
+            participants = participants.filter(f)
+        participants = participants.order_by(order()).limit(limit).offset(offset)
+
+    if 400 in participants:
+        return f"{participants}", 400
```

Comment:
What does this do?

Reply:
oops, this is no longer needed. removed in 57d248be0f491e0d8ad31f75add0169ad14b86d0